zfit/models/kde.py

```diff
@@ -0,0 +1,134 @@
+#  Copyright (c) 2020 zfit
+from typing import Union
+
+import numpy as np
+import tensorflow as tf
+import tensorflow_probability as tfp
+from tensorflow_probability.python import distributions as tfd
+
+from .dist_tfp import WrapDistribution
+from .. import z, ztypes
+from ..core.interfaces import ZfitData
+from ..util import ztyping
+from ..util.exception import OverdefinedError, ShapeIncompatibleError
+
+
+def bandwidth_rule_of_thumb(data, factor=0.9):
+    return tf.math.reduce_std(data) * tf.cast(tf.shape(data)[0], ztypes.float) ** (-1 / 5.) * factor
```

Comment:
Shouldn't the IQR be used as well as it is more robust for long-tailed and skew distributions.

```
return tf.minimum(tf.math.reduce_std(data), (tfp.stats.percentile(data, 75) - tfp.stats.percentile(data, 25)) / 1.34) * tf.cast(tf.shape(data)[0], ztypes.float) ** (-1 / 5.) * factor
```

Reply:
Yeah, it seems like. Do you refer to the one on [Wikipedia](https://en.wikipedia.org/wiki/Kernel_density_estimation)?

Reply:
Yes indeed.

Reply:
I've added that